.github/dependabot.yml

```diff
@@ -0,0 +1,23 @@
+version: 2
+updates:
+
+  # Maintain dependencies for GitHub Actions
+  - package-ecosystem: "github-actions"
```

Comment:
So we're going to upgrade all dependencies for Github Actions, not just security-related ones?

Reply:
security related are at the org level, this whole file was as per security team recommendation (as it stands, there is nothing to do, as the repo is just yaml)